packages/components/src/Disclosure/docs/inline.tsx

```diff
@@ -0,0 +1,20 @@
+import { Card, Disclosure, DisclosureHeader, DisclosurePanel, Div } from "@hopper-ui/components";
+
+export default function Example() {
+    return (
+        <Div width="100%" paddingX="core_320" paddingY="core_480">
+            <Card overflow="hidden">
```

Comment:
Writing another comment just so it doesnt get lost in the comment above, but i feel like this example is misrepresentative. 


It should be clearer what this variant is. Because at the moment, with the card, it look s100% like the examples above